src/lib/chat/index.ts

```diff
@@ -138,6 +139,10 @@ export class Chat {
     return this.client.fetchConversationsWithUsers(users);
   }
 
+  async markRoomAsRead(roomId: string, userId?: string): Promise<void> {
```

Comment:
why is the `userId` optional here?

Reply:
The Id is required in the sendbird client method.